applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.cpp

```diff
@@ -327,26 +328,19 @@ void UPwBaseElement<TDim, TNumNodes>::CalculateDampingMatrix(MatrixType&
     const unsigned int N_DOF = this->GetNumberOfDOF();
 
     // Compute Mass Matrix
-    MatrixType MassMatrix(N_DOF, N_DOF);
-
-    this->CalculateMassMatrix(MassMatrix, rCurrentProcessInfo);
+    MatrixType mass_matrix(N_DOF, N_DOF);
+    this->CalculateMassMatrix(mass_matrix, rCurrentProcessInfo);
 
     // Compute Stiffness matrix
-    MatrixType StiffnessMatrix(N_DOF, N_DOF);
-
-    this->CalculateMaterialStiffnessMatrix(StiffnessMatrix, rCurrentProcessInfo);
+    MatrixType stiffness_matrix(N_DOF, N_DOF);
+    this->CalculateMaterialStiffnessMatrix(stiffness_matrix, rCurrentProcessInfo);
 
     // Compute Damping Matrix
```

Comment:
And the same thing applies to this comment.

Reply:
removed